Specify UTC and fix typo on schedule edit page

Arithmomaniac opened the pull request:
- Ensure whenever "Last/Next Execution Time" is specified in the UI, it is noted that the time is in UTC
- Fix and issue where dates were being "double-converted" to UTC, causing the next execution of a script to sometimes be skipped.

Commit:
Specify UTC and fix typo on schedule edit page

Commit:
fix: prevent pre-emptive conversion of schedule time to UTC

Commit:
Specify that times are UTC on the script list page

brycekbargar commented on Milyli.ScriptRunner.Web/Views/JobSchedule/EditSchedule.cshtml:
```diff
@@ -18,9 +18,9 @@
-                    <label>Exectues At</label>
+                    <label>Last Execution Time (UTC)</label><span data-bind="text : LastExecutionTimeString"></span><br />
+                    <label>Next Execution Time (UTC)</label> <span data-bind="text : NextExecutionTimeString"></span><br />
+                    <label>Executes At (UTC)</label>
```
This is actually not UTC as we're saving the hour in the db.

Commit:
fix: undo incorrect labellng of displayed times as UTC

While they are stored in the DB as UTC, they are always rendered in local server time.
